src/index.js

```diff
@@ -163,30 +180,23 @@ exports.init = (Implementation) => {
   new SessionRoute(app, opts).perform();
 
   // Init
-  const absModelDirs = opts.modelsDir ? path.resolve('.', opts.modelsDir) : undefined;
-  requireAllModels(Implementation, absModelDirs)
-    .then((models) => {
-      integrator = new Integrator(opts, Implementation);
-
-      return Schemas.perform(Implementation, integrator, models, opts)
-        .then(() => {
-          let directorySmartImplementation;
+  buildSchema(Implementation)
+    .then(() => {
+      let directorySmartImplementation;
 
-          if (opts.configDir) {
-            directorySmartImplementation = path.resolve('.', opts.configDir);
-          } else {
-            directorySmartImplementation = `${path.resolve('.')}/forest`;
-          }
+      if (opts.configDir) {
+        directorySmartImplementation = path.resolve('.', opts.configDir);
+      } else {
+        directorySmartImplementation = `${path.resolve('.')}/forest`;
+      }
 
-          if (fs.existsSync(directorySmartImplementation)) {
-            return requireAllModels(Implementation, directorySmartImplementation);
-          }
-          if (opts.configDir) {
-            logger.error('The Forest modelsDir option you configured does not seem to be an existing directory.');
-          }
-          return [];
-        })
-        .thenReturn(models);
```

Comment:
Here, in the old code, it returns models from `absModelDirs`.
And in the new code it may return an empty array or models detected in the Smart directory.
The failing test might be a symptom of that.